environment-templates/vpc-env/v1/infrastructure/outputs.tf

```diff
@@ -0,0 +1,35 @@
+output "VpcId" {
+  value = module.vpc.vpc_id
+}
+
+output "PublicSubnetOneId" {
+  value = module.vpc.public_subnets[0]
+}
+
+output "PublicSubnetTwoId" {
+  value = module.vpc.public_subnets[1]
+}
+
+output "PrivateSubnetOneId" {
+  value = module.vpc.private_subnets[0]
+}
+
+output "PrivateSubnetTwoId" {
+  value = module.vpc.private_subnets[1]
+}
+
+output "VpcDefaultSecurityGroupId" {
+  value = module.vpc.default_security_group_id
+}
+
+output "VpcConnectorArn" {
+  value = aws_apprunner_vpc_connector.connector.id
+}
+
+output "SnsTopicName" {
+  value = aws_sns_topic.ping_topic.name
+}
+
+output "SnsRegion" {
+  value = local.region
+}
```

Comment:
You are missing [SnsTopicArn](https://github.com/syedsaadahmed1/aws-proton-sample-templates/blob/main/environment-templates/vpc-env/v1/infrastructure/cloudformation.yaml#L206)